understanding/22/hidden-controls.html

```diff
@@ -2,32 +2,40 @@
 <html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
    <meta charset="UTF-8"></meta>
-   <title>Understanding Hidden Controls</title>
+   <title>Understanding Visible Controls</title>
    <link rel="stylesheet" type="text/css" href="../../css/editors.css" class="remove"></link>
 </head>
 <body>
-   <h1>Understanding Hidden Controls</h1>
+   <h1>Understanding Visible Controls</h1>
    
     <section id="status" class="advisement">
         <h2>Status</h2>
         <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
     </section>
 
-
    <section class="remove">
    
-        <h2>Hidden Controls Success Criteria Text</h2>
+        <h2>Visible Controls Success Criteria Text</h2>
         <blockquote>
-        <p class="conformance-level">AA</p>
-        
-		<p>Controls needed to progress or complete a <a>process</a> are visible at the time they are needed without requiring pointer hover or keyboard focus, 
-		or a mechanism is available to make them persistently visible.</p>
+    
+         <p class="conformance-level">AA</p>
+        <p class="change">New</p>
+         
+         <p>Information needed to identify user interface components is visible without requiring pointer hover or keyboard focus, except when:</p>
+         <ul>
+             <li>Hiding the information needed to identify the control is essential;</li>
```

Comment:
struggling to come up with a rationale for when it's "essential". an example would help.